packages/accounts-base/accounts_server.js

```diff
@@ -1171,7 +1202,8 @@ Ap.config = function (options) {
 function setExpireTokensInterval(accounts) {
   accounts.expireTokenInterval = Meteor.setInterval(function () {
     accounts._expireTokens();
-    accounts._expirePasswordResetTokens();
+    accounts._expirePasswordResetTokens('reset');
```

Comment:
Looks like this didn't get updated; this seems to be passing the wrong param.  I believe it should be:

`accounts._expirePasswordResetTokens();`

?